signalfx/rest.py

```diff
@@ -427,3 +468,124 @@ def delete_detector(self, detector_id, **kwargs):
         resp.raise_for_status()
         # successful delete returns 204, which has no response json
         return resp
+
+    # Functionality related to charts
+    def get_chart(self, chart_id, **kwargs):
+        """Retrieve the single (v2) chart matching the given id.
+
+        Returns:
+            A (v2) chart as a JSON object (usable like a dictionary).
+        """
+        return self._get_object_by_id(self._CHART_ENDPOINT_SUFFIX, chart_id)
+
+    def get_charts(self, name=None, tags=None, batch_size=128, **kwargs):
+        """Retrieve all (v2) charts matching the given name; all (v2) charts up to batch_size otherwise.
```

Comment:
It's not "up to batch_size". It'll be all v2 charts, paging through them in pages of batch_size elements.